examples/BZNSYP/local/phn2id

```diff
@@ -0,0 +1,319 @@
+<pad> 0
```

Comment:
How can we get the file? It's better we checkin the scripts which gernerate phn2id.

Reply:
"phn2id" is from aishell3, but BZNSYP has some special phoneme which not in aishell3. Thus, I add these special phoneme to "phn2id". The code of "examples/BZNSYP/local/parse_data.py line 53" will check if all phonemes exist in "phn2id".